plugins/modules/database/misc/redis_info.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: redis_info
+short_description: Gather information about Redis servers
+description:
+- Gathers information and statistics about Redis servers.
+options:
+  login_host:
+    description:
+    - The host running the database
```

Comment:
```suggestion
    - The host running the database.
```
(Same for the next ones. `description`s should always be complete sentences, except `short_description`.)